src/grisp_onewire.erl

```diff
@@ -66,7 +94,42 @@ reset() ->
         Any -> error({invalid_status, Any})
     end.
 
-%% Spec: | S | AD,0 ‖ A ‖ WCFG ‖ A ‖ <byte> ‖ A ‖ Sr | AD,1 ‖ A | <byte> ‖ A\ | P |
+%% @doc Write configuration into 1-Wire master register.
+%%
+%% Command code: d2h.
+%%
+%% This function can only be used inside of a {@link transaction/1} call.
+%%
+%% The default configuration is 0, i.e., all three configurable bits set to 0.
```

Comment:
Let's use the EUnit way to render things in monospace for constants:  `` `0' ``.